vhdbuilder/scripts/linux/mariner/tool_installs_mariner.sh

```diff
@@ -131,3 +131,33 @@ enableMarinerKata() {
     # dracut to generate an initrd for the nested VM using binaries from the Mariner host OS.
     systemctlEnableAndStart kata-osbuilder-generate
 }
+
+# Explicitly configure containerd systemd service on Mariner AKS to maintain consistent
+# settings with the containerd.service file previously deployed during cloud-init.
+# Additionally set LimitNOFILE to the exact value "infinity" means on Ubuntu, eg "1048576".
+# This is because infinity on Mariner 2.0 means "1073741816", which is known to cause issues in containers.
+setContainerdServiceConfig() {
```

Comment:
would be nice to have the actual artifact as a file, I know it's more annoying to add to the packer.json.

I did this for a bunch of refactoring recently in CSE, but next step is move all those directly into VHD

Reply:
Sure thing, moving it into the packer file copy instead.